src/component-library/TokenStack/TokenStack.style.tsx

```diff
@@ -0,0 +1,25 @@
+import styled from 'styled-components';
+
+import { Flex } from '../Flex';
+import { theme } from '../theme';
+import { IconSize } from '../utils/prop-types';
+
+type TokenOffset = 'none' | '1/2' | '1/3' | '1/4' | '1/5';
```

Comment:
Do we have use cases for all of these values? It's not a critical issue, but I'd prefer to limit the offset to values we use (would of course be very easy to add more in if required)

Reply:
The only one I would consider discarding would be the `1/5`.

`none` - because in some parts of the design Jay removes the offset
`1/2` - because when we get to a pool with 4 tokens, we need to squish them into an `Adornment` in our `TokenInput` like this: 
![image](https://user-images.githubusercontent.com/43269067/211044033-30008c8d-c44f-4414-984a-af6e9ebc1e4e.png)
`1/3` - our default
`1/4`- a more loose alternative to the default.
1/5` - we can possibly discard.

Reply:
That makes sense - I would remove 1/5 if we're not using it

Reply:
I would also suggest reviewing this with Jay at some point (not urgent though), as the issue with the adornment sounds like a UX issue, and it probably has a UX fix :)